lib/slax_web/websockets/issue.ex

```diff
@@ -10,7 +10,7 @@ defmodule SlaxWeb.Issue do
   def handle_event(%{"bot_id" => bot_id}) when not is_nil(bot_id), do: nil
 
   def handle_event(%{"thread_ts" => ts, "channel" => channel, "text" => text, "type" => "message"}) do
-    with issues when issues != [] <- Regex.scan(~r{([\w-]+/)?([\w-]+)?(#[0-9]+)}, text) do
+    with issues when issues != [] <- Regex.scan(~r{([\w-]+/)?([\w-]+)(#[0-9]+)}, text) do
```

Comment:
maybe we can write tests for this?

Reply:
Yes, Jason mentioned that, working on them rn I just wanted to get this in now that I have internet again